ui/src/utils/index.ts

```diff
@@ -1,5 +1,12 @@
 import { Pod, PodDetail, ResourceUsage } from "./models/pods";
 
+export function getBaseHref(): string {
+  if (process.env.REACT_APP_BASE_HREF) {
+    return process.env.REACT_APP_BASE_HREF
+  }
+  return "";
```

Comment:
Should it return `"/"`?

Reply:
You are right, if we want ppl to configure `/app` (no ending `/`), then the default should be empty string.